src/lambda_functions/get_prices_lambda/scraper.py

```
@@ -561,202 +575,190 @@
+            logger.error(f"Config file not found: {config_path}")
+            raise ConfigError(f"Configuration file not found: {config_path}")
+        except yaml.YAMLError as e:
+            logger.error(f"Error parsing config file: {str(e)}")
+            raise ConfigError(f"Invalid YAML format: {str(e)}")
+        except Exception as e:
+            logger.error(f"Error loading config: {str(e)}")
+            raise ConfigError(f"Failed to load configuration: {str(e)}")
+        
+    def _handle_error(self, error: Exception, url: str = "unknown", severity: ErrorSeverity = ErrorSeverity.MEDIUM) -> None:
+        """エラーハンドリングの共通処理"""
+        error_id = str(uuid.uuid4())
```

Ruff (0.8.2)

Local variable `error_id` is assigned to but never used

Remove assignment to unused variable `error_id`

(F841)